metricbeat/module/system/raid/blockinfo/parser.go

```diff
@@ -0,0 +1,202 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package blockinfo
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/logp"
+)
+
+var debugf = logp.MakeDebug("system-raid")
```

Comment:
I think this should be `system.raid`.
```suggestion
var debugf = logp.MakeDebug("system.raid")
```
I see we have it as `system-network` in the network metricset, we should probably change it there too.

Reply:
Ah, missed that! I can put in a PR to fix the network one. I was actually going to ask if creating a new logger was the 'right' way to do it.